README.md

```diff
@@ -14,7 +14,7 @@ If you have any question please [reach out to the team in our Gitter chat](https
 
 ## 2 - Create your first project!
 
-1. `yarn create comit-app new <your-app-name>`,
+1. `yarn create comit-app <your-app-name>`,
 2. `yarn create comit-app start-env` to start blockchain and COMIT nodes,
```

Comment:
I think this line can/should go away now.

Reply:
Which line? Starting the environment? Should there be a `yarn start-env` to run the example a bit below then?